packages/ripple/index.tsx

```diff
@@ -147,18 +150,18 @@ export function withRipple <
     this.foundation.init();
   };
 
-  createAdapter: MDCRippleAdapter = (surface: Surface, activator?: Activator) => {
+  createAdapter = (surface: Surface, activator?: Activator): MDCRippleAdapter => {
     return {
       browserSupportsCssVars: () => util.supportsCssVariables(window),
-      isUnbounded: () => this.props.unbounded,
+      isUnbounded: () => (this.props.unbounded || false) as boolean,
```

Comment:
shouldn't you be able to do `isUnbounded: () => this.props.unbounded!,` here. `defaultProps` are set here, so `!` should be valid. Same goes for isSurfaceDisabled below.